src/components/tx/TxSecurityWarnings/index.tsx

```diff
@@ -15,10 +18,26 @@ export const TxSecurityWarnings = ({ safeTx }: { safeTx: SafeTransaction | undef
         redefineScanResult && (
           <>
             <BalanceChanges balanceChange={redefineScanResult.payload?.balanceChange} />
-            {redefineScanResult?.payload?.issues?.map((issue) => (
+            {redefineScanResult.payload?.issues?.map((issue) => (
               <SecurityHint key={issue.category} severity={issue.severity} text={issue.description.short} />
             ))}
-            <SecurityWarning severity={redefineScanResult?.severity} />
+            <SecurityWarning severity={redefineScanResult.severity} />
```

Comment:
We should sync after merging the PR or before on how to consolidate both results into one component.
They should appear in the same block.

Reply:
I agree. `TxSecurityWarnings` is just for demonstrative purposes at the moment though.